{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"InternalError"}],"languages":["swift"],"platforms":["iOS"]}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/processout\/processoutexception\/internalerror"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/com.processout.processout-ios\/documentation\/ProcessOut\/ProcessOutException\/InternalError","interfaceLanguage":"swift"},"kind":"symbol","metadata":{"role":"symbol","title":"ProcessOutException.InternalError","roleHeading":"Case","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"InternalError"}],"symbolKind":"case","externalID":"s:10ProcessOut0aB9ExceptionO13InternalErroryA2CmF","modules":[{"name":"ProcessOut"}],"platforms":[]},"hierarchy":{"paths":[["doc:\/\/com.processout.processout-ios\/documentation\/ProcessOut","doc:\/\/com.processout.processout-ios\/documentation\/ProcessOut\/ProcessOutException"]]},"deprecationSummary":[{"type":"paragraph","inlineContent":[{"type":"text","text":"Use POFailure instead."}]}],"references":{"doc://com.processout.processout-ios/documentation/ProcessOut/ProcessOutException":{"deprecated":true,"role":"symbol","title":"ProcessOutException","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"ProcessOutException"}],"abstract":[],"identifier":"doc:\/\/com.processout.processout-ios\/documentation\/ProcessOut\/ProcessOutException","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"ProcessOutException"}],"url":"\/documentation\/processout\/processoutexception"},"doc://com.processout.processout-ios/documentation/ProcessOut/ProcessOutException/InternalError":{"role":"symbol","title":"ProcessOutException.InternalError","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"InternalError"}],"abstract":[],"identifier":"doc:\/\/com.processout.processout-ios\/documentation\/ProcessOut\/ProcessOutException\/InternalError","kind":"symbol","type":"topic","deprecated":true,"url":"\/documentation\/processout\/processoutexception\/internalerror"},"doc://com.processout.processout-ios/documentation/ProcessOut":{"role":"collection","title":"ProcessOut","abstract":[],"identifier":"doc:\/\/com.processout.processout-ios\/documentation\/ProcessOut","kind":"symbol","type":"topic","url":"\/documentation\/processout"}}}